### PR TITLE
C#: Update all nuget packages

### PR DESCRIPTION
**Pull request by hvitved**
Update all nuget packages to the latest version.

### CODE REVIEW

File: csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs [hunk 1]
```diff
@@ -161,6 +162,9 @@ public static bool OperatorSymbol(string methodName, out string operatorName)
                 case "op_False":
                     operatorName = "false";
                     break;
+                case var @checked when Regex.IsMatch(@checked, "^op_Checked.*"):
+                    operatorName = @checked;
+                    break;
```
Comment (tamasvajk) [inline csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs:167]: Why is this change needed? Are there any tests that cover it? `checked +` might be another operator name that we could use. That would match better the unchecked `+`.

Comment (hvitved) [reply csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs:167]: Indeed; we should use `checked +` instead. Apparently some existing tests exercise this; probably because checked operators exist in library code.

File: csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs [hunk 2]
```diff
@@ -166,6 +166,13 @@ public static bool OperatorSymbol(string methodName, out string operatorName)
                     operatorName = @checked;
                     break;
                 default:
+                    var match = Regex.Match(methodName, "^op_Checked(.*)$");
+                    if (match.Success)
+                    {
+                        OperatorSymbol("op_" + match.Groups[1], out var uncheckedName);
+                        operatorName = "checked " + uncheckedName;
+                        break;
+                    }
```
Comment (tamasvajk) [inline csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs:175]: Do we need this in the `default` branch? Why wasn't this added in the `var @checked when Regex.IsMatch(@checked, "^op_Checked.*"):` case? Or maybe the `var @checked when ...` could be removed now.

Comment (tamasvajk) [reply csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs:175]: But isn't the below redundant?
```
case var @checked when Regex.IsMatch(@checked, "^op_Checked.*"):
                    operatorName = @checked;
                    break;
```

Comment (hvitved) [reply csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs:172]: Ah 🤦 I thought I had removed it; is removed now.

**Review by tamasvajk — APPROVED** [review]
